src/Network/Network/NetworkSecurityPerimeter.Autorest/README.md

```diff
@@ -0,0 +1,187 @@
+<!-- region Generated -->
+# Az.NetworkSecurityPerimeter
+This directory contains the PowerShell module for the NetworkSecurityPerimeter service.
+
+---
+## Status
+[![Az.NetworkSecurityPerimeter](https://img.shields.io/powershellgallery/v/Az.NetworkSecurityPerimeter.svg?style=flat-square&label=Az.NetworkSecurityPerimeter "Az.NetworkSecurityPerimeter")](https://www.powershellgallery.com/packages/Az.NetworkSecurityPerimeter/)
+
+## Info
+- Modifiable: yes
+- Generated: all
+- Committed: yes
+- Packaged: yes
+
+---
+## Detail
+This module was primarily generated via [AutoRest](https://github.com/Azure/autorest) using the [PowerShell](https://github.com/Azure/autorest.powershell) extension.
+
+## Module Requirements
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 2.7.5 or greater
+
+## Authentication
+AutoRest does not generate authentication code for the module. Authentication is handled via Az.Accounts by altering the HTTP payload before it is sent.
+
+## Development
+For information on how to develop for `Az.NetworkSecurityPerimeter`, see [how-to.md](how-to.md).
+<!-- endregion -->
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+require:
+# readme.azure.noprofile.md is the common configuration file
+# - ../../../../tools/SwaggerCI/readme.azure.noprofile.md
+  - $(this-folder)/../../../readme.azure.noprofile.md
+input-file:
+# You need to specify your swagger files here.
+  - https://github.com/kaushal087/azure-rest-api-specs/blob/nsp_powershell/specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkSecurityPerimeter.json
```

Comment:
```suggestion
  - https://github.com/kaushal087/azure-rest-api-specs/blob/nsp_powershell/specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkSecurityPerimeter.json
branch: commit-hash
```
Add commit hash so that the generated code will always be the same no matter how your branch updates.

Reply:
Added commit hash link.